qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -7,7 +7,7 @@ def bundle_url_from_version(version)
     when true    then '""'      # Use bucket version
     when false   then '""'      # Don't install (non-upgrade pipelines, for example)
     when /:\/\// then version # URL
-    else "https://((s3-config-bucket-sles)).s3.amazonaws.com/((s3-config-prefix-sles))#{ERB::Util.url_encode(version)}.zip"
+    else "https://s3.amazonaws.com/((s3-config-bucket-sles))/master/#{ERB::Util.url_encode(version)}.zip"
```

Comment:
AWS has deprecated the `s3.amazonaws.com` version: https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/

You're changing common code, which means `s3-config-prefix-sles` will basically never be used when deploying non-upgrade pipelines at a specific version.  We need a different fix here (to pass down the correct prefix, possibly as part of the version).

Reply:
Ok. I have created card for this https://jira.suse.com/browse/CAP-365